tensorrt_llm/_torch/disaggregation/native/bounce/gather_scatter.py

```
@@ -0,0 +1,232 @@
+            devt[n : 2 * n],
+            devt[2 * n : 3 * n],
+            n,
+            BLOCK=_BLOCK,
+            num_warps=_NUM_WARPS,
+        )
+    return True
+
+
+def _copy_frags(pairs, sizes: np.ndarray, stream) -> None:
+    """Fallback (no Triton/GPU or unaligned): one async D2D copy per fragment; pairs yields (dst_ptr, src_ptr)."""
+    for (dst, src), n in zip(pairs, sizes):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)